website/thaliawebsite/menus.py

```diff
@@ -28,6 +28,7 @@
             },
             {"title": _("Become a Member"), "name": "registrations:index"},
             {"title": _("Alumni"), "name": "events:alumni"},
+            {"title": _("Trust contacts"), "name": "singlepages:trust-contacts",},
```

Comment:
Would 'student well-being' not be a better term for the whole thing?

Reply:
@SnevelRetlaw

Reply:
yes, we wanted to call it something like "student help" or "student well-being", so you can change it to "student well-being"

Reply:
I put it back under "education" and did the rename